mobify/platform-scaffold
# WEBDATA-240 Inline sign in and cart merge modal

**JIRA**: https://mobify.atlassian.net/browse/WEBDATA-240
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Added cart merge items modal
- Added inline sign in fields for checkout
- Removed checkout header sign in link

## Todos

## How to test-drive this PR
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Sign in
- Add potions to your cart
- Sign out
- Add more potions to your cart
- Checkout
- sign in during checkout
- Verify after you signed in, a modal pops up and warns that there are items added to the cart from previous sessions
![screen shot 2017-08-31 at 11 38 36 am](https://user-images.githubusercontent.com/10948652/29939529-f7e0a48e-8e40-11e7-8814-fd77fc387498.png)
